src/js/helpers/general.js

```diff
@@ -14,6 +15,15 @@ function setTuningData(tuning) {
   jQuery('#txt_tuning_data').val(tuning)
 }
 
+function isTuningDataAvailable(alertIfNot = false, alertMsg = 'No tuning data.') {
+  trimSelf('#txt_tuning_data')
+  if (isEmpty(jQuery('#txt_tuning_data').val())) {
+    if (alertIfNot) alert(alertMsg)
```

Comment:
Please add `alert` to the globals on the beginning of the file, so that the linters know where it comes from.